### PR TITLE
ci(jenkins): Install VS2019Professional in the Windows CI Workers

### PR DESCRIPTION
**Pull request by v1v**
Use VS2019Professional

### CODE REVIEW

File: .ci/windows/msbuild-tools.ps1 [hunk 1]
```diff
@@ -3,6 +3,6 @@
 #
 
 # Install visualstudio2019buildtools
-& choco install visualstudio2019buildtools -m -y --no-progress --force -r --version=16.4.0.0
-& choco install visualstudio2019enterprise -m -y --no-progress --force -r --version=16.4.0.0 --package-parameters "--includeRecommended --includeOptional"
-& choco install visualstudio2019-workload-netweb -m -y --no-progress --force -r --version=1.0.0
+& choco install visualstudio2019buildtools -m -y --force --no-progress -r
+& choco install visualstudio2019professional -m -y --force --no-progress -r --package-parameters "--includeRecommended --includeOptional"
+& choco install visualstudio2019-workload-netweb -m -y --no-progress -r --version=1.0.0
```
Comment (gregkalapos) [inline .ci/windows/msbuild-tools.ps1]: Let's add `--force` here, too. Maybe that'd help!

Comment (gregkalapos) [reply .ci/windows/msbuild-tools.ps1]: Oh, I see you already have it in https://github.com/elastic/apm-agent-dotnet/pull/759 - that's good.

**Comment by v1v** [comment]
Yep, I did trigger the build manually to double-check everything is in good shape. Do you prefer to merge this after #754.

**Comment by gregkalapos** [comment]
> Do you prefer to merge this after #754.

No, let's merge this if everyone thinks this is ok, I don't think we need to wait for #754. From my side, this is ok to merge now.

**Pull request merged**